internal/common/filespec.go

```diff
@@ -47,6 +48,11 @@ func (this *FileSpec) Validate() error {
 		if this.Path == "" {
 			return fmt.Errorf("A path FileSpec cannot have an empty path.")
 		}
+
+		_, err := filepath.Glob(this.Path)
+		if err != nil {
+			return fmt.Errorf("Invalid glob pattern '%s': '%w'.", this.Path, err)
```

Comment:
This error will be confusing for people who are not using (or don't know about) globs.
They didn't select a "glob-type" FileSpec.